packages/@uppy/aws-s3/types/index.d.ts

```diff
@@ -10,8 +9,20 @@ export interface AwsS3UploadParameters {
     headers?: { [type: string]: string }
 }
 
-export interface AwsS3Options extends AwsS3MultipartOptions {
-    /** @deprecated future versions of this plugin will use the Expires value from the backend */
+export interface AwsS3Options {
```

Comment:
There I removed the inheritance from `AwsS3MultipartOptions`. Its possible that they should both inherit from a `BaseS3Options` but I did not do that here.